src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4121,6 +4142,15 @@ private void RegisterPartial(string format, params object[] values)
 			_partials.Add(format.InvariantCultureFormat(values));
 		}
 
+		private void RegisterXBindTryGetDeclaration(string declaration)
+		{
+			if (_fileDefinition.FilePath.Contains("When_xLoad_DataTemplate_In_ResDict_Global", StringComparison.Ordinal))
+			{
+				//Debugger.Launch();
+			}
```

Comment:
Remove

Reply:
@jeromelaban Already removed in the last commit